src/test/java/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,67 @@
+package strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashMap;
+import java.util.Map;
+import model.FruitTransaction;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import strategy.impl.BalanceTransactionImpl;
+import strategy.impl.OperationStrategyImpl;
+import strategy.impl.PurchaseTransactionImpl;
+import strategy.impl.ReturnTransactionImpl;
+import strategy.impl.SupplyTransactionImpl;
+
+class OperationStrategyTest {
```

Comment:
OperationStrategyTest doesn't test the OperationStrategyImpl class. 

Should be used `OperationStrategyImpl` in test cases.